tests/integration/test_image_filters.py

```diff
@@ -120,6 +120,84 @@ def test_rgb_to_hed_raises_exception_on_gs_image():
     assert str(err.value) == "Input image must be RGB."
 
 
+def test_hematoxylin_channel_filter_with_rgb_image():
+    img = RGB.TCGA_LUNG_RGB
+    expected_value = load_expectation(
+        "pil-images-rgb/tcga-lung-rgb-hematoxylin-channel", type_="png"
+    )
+
+    hematoxylin_img = imf.hematoxylin_channel(img)
+
+    np.testing.assert_array_almost_equal(
+        np.array(hematoxylin_img), np.array(expected_value)
+    )
+    assert (
+        np.unique(np.array(ImageChops.difference(hematoxylin_img, expected_value)))[0]
+        == 0
+    )
```

Comment:
can you try to reduce `hematoxylin_img` in order to not have this odd carriage return in the assert statement?